tests/main/security-seccomp/task.yaml

```diff
@@ -0,0 +1,64 @@
+summary: Check basic seccomp rules
+
+# This test verifies that:
```

Comment:
Nitpick because nothing checks for this but usually we do comments like this inside a "details" section:
```
details: |
  loads
  of
  text
```